tools/accuracy_checker/accuracy_checker/annotation_converters/librispeech.py

```diff
@@ -0,0 +1,1614 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from pathlib import Path
+import numpy as np
+import re, math, wave, time, codecs
+
+from ..representation import CharacterRecognitionAnnotation
+from ..config import NumberField, StringField, PathField, BoolField
+from .format_converter import DirectoryBasedAnnotationConverter
+from .format_converter import ConverterReturn
+from scipy.fftpack import dct
+from scipy.signal import stft
+
+class LibrispeechConverter(DirectoryBasedAnnotationConverter):
+    __provider__ = 'librispeech'
+    annotation_types = (CharacterRecognitionAnnotation, )
+    HZ2MEL = 1127.0
+
+    # @classmethod
```

Comment:
please do not forget to remove debug comments

Reply:
fixed